superset-frontend/spec/fixtures/mockNativeFilters.ts

```diff
@@ -72,33 +73,34 @@ export const nativeFilters: NativeFiltersState = {
       isInstant: true,
     },
   },
-  filtersState: {
-    crossFilters: {},
-    ownFilters: {},
-    nativeFilters: {
-      'NATIVE_FILTER-e7Q8zKixx': {
-        id: 'NATIVE_FILTER-e7Q8zKixx',
-        extraFormData: {
-          append_form_data: {
-            filters: [
-              {
-                col: 'region',
-                op: 'IN',
-                val: ['East Asia & Pacific'],
-              },
-            ],
-          },
-        },
-        currentState: {
-          value: ['East Asia & Pacific'],
+};
+
+export const dataMask: DataMaskStateWithId = {
```

Comment:
Could we make this name more descriptive?

Reply:
for example? I also not sure that it's good to add to complexity to name, because in the code we also add prefixes/postfixes, function name to this basic name...